kiva/quartz/ABCGI.pyx

```diff
@@ -809,7 +809,11 @@ cdef class CGContext:
             constants.ITALIC: 'italic',
             constants.BOLD_ITALIC: 'bold italic',
         }[font.is_bold() | font.style]
-        self.select_font(font.face_name, font.size, style=style)
+        if font.face_name:
+            name = font.face_name
+        else:
+            name = font.findfontname()
```

Comment:
It feels like this is something the kiva `Font` class should be managing, but perhaps that's too fraught with all of the different backend implementations.

Reply:
Possibly.  I'd like to see `Font` to grow more like the CSS font specifiers where you can give a bunch of font face name suggestions, including generic ones, but that's a fairly big API change that would need to be managed.